src/toolbox/hardware_components/color_sensor.ts

```diff
@@ -22,7 +22,7 @@
 
 import * as ToolboxItems from '../items';
 
-export const TYPE_NAME = 'ColorRangeSensor';
+export const TYPE_NAME = 'color_range_sensor.ColorRangeSensor';
```

Comment:
Why are the type_names seemingly redundant?   Is it module followed by class?   Do we have modules that have more than one class?